openingd/dualopend.c

```diff
@@ -2408,9 +2435,40 @@ static u8 *opener_commits(struct state *state,
 		return NULL;
 	}
 
+	if (!amount_sat_to_msat(&their_msats, tx_state->accepter_funding)) {
+		status_failed(STATUS_FAIL_INTERNAL_ERROR,
+			      "Rounding error, can't convert accepter_funding %s"
```

Comment:
it's an overflow error, not a rounding error

Reply:
fixed